install/syndesis-dev.yml

```diff
@@ -1633,6 +1633,16 @@ objects:
   message: |-
     Syndesis upgrade process started
   objects:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
```

Comment:
Well, the upgrade pod currently creates a subdirectory in the volume using a timestamp, so that subsequent upgrades won't clash.

The operator is already instructed not to replace PVC, because replacing them results in loss of data, so if a PVC is already present, it will be used.

We can create a PVC in advance, but current 7.1 (and older) installations don't have it. So, even if we include it in standard installations, we also need to create it if not present when doing the actual upgrade.

In the non-operator case, the "manual operator" would need to create the pod and the PVC if not present. Upgrade without operator is a completely manual process iirc.

We can add the PVC to both locations (main template and sub-template), so we'll have less issues when upgrading future installations, wdyt?

Reply:
I see. Actually, I would add it in the main template for sure for the reason to fail fast if this is not possible.  Then we can leave it in the template for 7.2.0 and maybe remove it for 7.3.0 (because then it is supposed to be there anyway either because of a fresh install or because updated from 7.2 which will also have this PVC). 

So happy for now to have it in both spots.

Reply:
@kcbabo @gaughan Just a heads up, we are adding an additional persistent volume requirement for Fuse Online. AFAIR we have currently 3 PV a 1 GB as minimal requirement for Fuse Online (tbv), so this would increase to 4 and this would also affect the plans on OSO evaluation.

Not sure how the process is to propagate such requirement changes and how to deal with these from version to version, so my first attempt is to ping you directly ;-)

Reply:
I've added the claim also to the first installation in the meantime.

Reply:
Yes, it makes sense. I can do it.